src/encodings.jl

```diff
@@ -1,271 +1,384 @@
+    "JIS_X0212.1990-0", "JIS0208", "JISX0201-1976", "Johab", "JOHAB",
+    "JP", "JP-OCR-B", "JS", "JUS_I.B1.002", "KOI-7", "KOI-8", "KOI8",
+    "koi8-r", "KOI8-R", "KOI8-RU", "KOI8-T", "koi8-u", "KOI8-U",
+    "KOI8R", "KOI8U", "KOREAN", "ks_c_5601-1987", "KS_C_5601-1987",
+    "KS_C_5601-1989", "KSC_5601", "KSC5636", "KZ-1048", "L1", "L10",
+    "L2", "L3", "L4", "L5", "L6", "L7", "L8", "LATIN-9", "LATIN-GREEK",
+    "LATIN-GREEK-1", "LATIN1", "LATIN10", "LATIN2", "LATIN3", "LATIN4",
+    "LATIN5", "LATIN6", "LATIN7", "LATIN8", "LATIN9", "LATINGREEK",
+    "LATINGREEK1", "MAC", "MAC-CENTRALEUROPE", "MAC-CYRILLIC", "MAC-IS",
+    "MAC-SAMI", "MAC-UK", "MACARABIC", "MACCENTRALEUROPE", "MACCROATIAN",
+    "MACCYRILLIC", "MACGREEK", "MACHEBREW", "MACICELAND", "macintosh",
+    "MACINTOSH", "MACIS", "MACROMAN", "MACROMANIA", "MACTHAI", "MACTURKISH",
+    "MACUK", "MACUKRAINE", "MACUKRAINIAN", "MIK", "MS_KANJI", "MS-ANSI",
+    "MS-ARAB", "MS-CYRL", "MS-EE", "MS-GREEK", "MS-HEBR", "MS-MAC-CYRILLIC",
+    "MS-TURK", "MS50221", "MS51932", "MS932", "MS936", "MSCP1361",
+    "MSCP949", "MSMACCYRILLIC", "MSZ_7795.3", "MULELAO-1", "NAPLPS",
+    "NATS-DANO", "NATS-SEFI", "NATSDANO", "NATSSEFI", "NC_NC00-10",
+    "NC_NC00-10:81", "NC_NC0010", "NEXTSTEP", "NF_Z_62-010", "NF_Z_62-010_(1973)",
+    "NF_Z_62-010_1973", "NF_Z_62010", "NF_Z_62010_1973", "NO", "NO2",
+    "NS_4551-1", "NS_4551-2", "NS_45511", "NS_45512", "OS2LATIN1",
+    "OSF00010001", "OSF00010002", "OSF00010003", "OSF00010004", "OSF00010005",
+    "OSF00010006", "OSF00010007", "OSF00010008", "OSF00010009", "OSF0001000A",
+    "OSF00010020", "OSF00010100", "OSF00010101", "OSF00010102", "OSF00010104",
+    "OSF00010105", "OSF00010106", "OSF00030010", "OSF0004000A", "OSF0005000A",
+    "OSF05010001", "OSF10010001", "OSF10010004", "OSF10010006", "OSF10020025",
+    "OSF10020111", "OSF10020115", "OSF10020116", "OSF10020118", "OSF1002011C",
+    "OSF1002011D", "OSF10020122", "OSF10020129", "OSF100201A4", "OSF100201A8",
+    "OSF100201B5", "OSF100201F4", "OSF10020352", "OSF10020354", "OSF10020357",
+    "OSF10020359", "OSF1002035D", "OSF1002035E", "OSF1002035F", "OSF10020360",
+    "OSF10020364", "OSF10020365", "OSF10020366", "OSF10020367", "OSF1002036B",
+    "OSF10020370", "OSF1002037B", "OSF10020387", "OSF10020388", "OSF10020396",
+    "OSF100203B5", "OSF10020402", "OSF10020417", "PT", "PT154", "PT2",
+    "PTCP154", "R8", "R9", "RK1048", "ROMAN8", "ROMAN9", "RUSCII",
+    "SE", "SE2", "SEN_850200_B", "SEN_850200_C", "SHIFFT_JIS", "SHIFFT_JIS-MS",
+    "shift_jis", "SHIFT_JIS", "SHIFT_JISX0213", "shift-jis", "SHIFT-JIS",
+    "SJIS", "SJIS-MS", "SJIS-OPEN", "SJIS-WIN", "SS636127", "ST_SEV_358-88",
+    "STRK1048-2002", "T.61", "T.61-8BIT", "T.618BIT", "TCVN", "TCVN-5712",
+    "TCVN5712-1", "TCVN5712-1:1993", "THAI8", "TIS-620", "TIS620",
+    "TIS620-0", "TIS620.2529-1", "TIS620.2533-0", "TIS620.2533-1",
+    "TS-5881", "TSCII", "TURKISH8", "UCS-2", "UCS-2-INTERNAL", "UCS-2-SWAPPED",
```

Comment:
I'm not even sure what they mean by some of these Unicode related ones, INTERNAL / SWAPPED may be what I was getting at, with native vs opposite.